assets/app/features/upgrade/i18n/en.json

```diff
@@ -61,19 +61,19 @@
             },
             "database-configuration": {
                 "name": "",
-                "title": "Enter Credentials and/or Information to Create or Connect to an OpenStack PostgreSQL Database for the OpenStack Database Migration.",
+                "title": "Enter Credentials and/or Information to Create or Connect to an OpenStack PostgreSQL Server for the OpenStack Database Migration.",
                 "navigation-tabs": {
-                    "create-database": "Create New Database",
-                    "connect-database": "Connect to an Existing Database"
+                    "create-server": "Create New Server",
```

Comment:
Im guessing the refinement of this will come later, because there is a huge difference in what it says and in what it *does*.